.github/workflows/recreate-qa-database.yml

```diff
@@ -0,0 +1,76 @@
+name: Replace Postgres Database
+
+on:
+  workflow_dispatch:
+    inputs:
+      environment:
+        description: Environment
+        default: qa
+        required: true
+
+env:
+ DOCKER_REPOSITORY: ghcr.io/dfe-digital/teaching-vacancies
+
+jobs:
+
+  deploy-app:
+    name: Deploy master image tag to ${{ github.event.inputs.environment }}
+    runs-on: ubuntu-20.04
+    steps:
+
+    - uses: DFE-Digital/github-actions/turnstyle@master
+      timeout-minutes: 20
```

Comment:
As discussed, can you replacing the job property timeout-minutes with abort-after-seconds step property

Reply:
And change also: https://github.com/DFE-Digital/teaching-vacancies/blob/master/.github/workflows/deploy_app.yml#L20